daffodil-runtime1-unparser/src/main/scala/org/apache/daffodil/unparsers/runtime1/PackedBinaryUnparserTraits.scala

```diff
@@ -94,9 +94,9 @@ abstract class PackedBinaryDecimalBaseUnparser(
     val bigDec = number.asInstanceOf[JBigDecimal]
     if (bigDec.movePointRight(binaryDecimalVirtualPoint).scale != 0) {
       e.schemaDefinitionError(
-        "Decimal point of number '%s' does not match the binaryVirtualDecmialPoint: %d",
+        "Decimal point of number '%s' does not match the binaryVirtualDecmialPoint: %s",
         bigDec,
-        binaryDecimalVirtualPoint,
+        binaryDecimalVirtualPoint.toString,
```

Comment:
For  scala/java reasons SDE methods taking varargs had to change to take AnyRef, not Any, so some places like this that were using primitive types had to switch to boxed or strings. 

However, I should review if this change remains necesary, and revert if possible